.github/workflows/manual-deploy-testnet-l2.yml

```diff
@@ -355,15 +348,8 @@ jobs:
   deploy-faucet:
     runs-on: ubuntu-latest
     needs:
-      - update-loadbalancer
-      - deploy-l2-contracts
+      - check-obscuro-is-healthy
     steps:
-      - name: 'Trigger Faucet deployment on workflow dispatch'
-        if: ${{ github.event_name == 'workflow_dispatch' }}
-        run: |
-          curl -XPOST -H "Authorization: Bearer ${{ secrets.GH_TOKEN }}" -H "Accept:application/vnd.github" -H "Content-Type:application/json" https://api.github.com/repos/obscuronet/faucet/actions/workflows/manual-deploy-${{ github.event.inputs.testnet_type }}-faucet.yml/dispatches --data '{"ref": "main" }'
-
-      - name: 'Trigger Faucet deployment on schedule'
-        if: ${{ github.event_name == 'schedule' }}
+      - name: 'Trigger Faucet deployment'
         run: |
-          curl -XPOST -H "Authorization: Bearer ${{ secrets.GH_TOKEN }}" -H "Accept:application/vnd.github" -H "Content-Type:application/json" https://api.github.com/repos/obscuronet/faucet/dispatches --data '{ "event_type": "l2_dev_deployment", "client_payload": { "ref": "main", "env": "dev-testnet" }'
+          curl -XPOST -H "Authorization: Bearer ${{ secrets.GH_TOKEN }}" -H "Accept:application/vnd.github" -H "Content-Type:application/json" https://api.github.com/repos/obscuronet/faucet/dispatches --data '{ "event_type": "${{ github.event.inputs.testnet_type/-/_ }}_deployed", "client_payload": { "ref": "${{ github.ref }}", "env": "testnet", "event": "${{ github.event_name }}" }'
```

Comment:
Does the "env" field need to be variable too? To say like `"env": "${{github.event.inputs.testnet_type}}"`

Reply:
yes, tbh easiest to just remove that anyway